### PR TITLE
Remove jekyll-last-modified-at

### PR DESCRIPTION
**Pull request by bronzdoc**
1. The plugin doesn't work on Heroku, because the plugin picks the date from git but it seems heroku deletes the .git directory as part of their process before building the project, so the plugin doesn't have access to use the git dates.

2. If no git reference of is found the plugin will try to get the mtime of the file, but since heroku process involves moving files to an app directory all the files mtime will be set to the time this action happens, leading     to putting all the blogposts last_modified_at field to the same time the blog was deployed.

I'm removing the use of this plugin and the last_modified_at functionality since it doesn't work with our current deployment process.

We will have to find a way to manage it our selves or find another plugin.

### CODE REVIEW

**Comment by bronzdoc** [comment]
ping @etagwerker @lubc

**Comment by bronzdoc** [comment]
@lubc since blog posts are static files, I was thinking on having some kind of `git hook` that writes to a "blogpost_upated.json" everytime we do a commit.  The file would be a key-value store that matches `"blogspost file name" => { last_modified_at: "Tue Sep 24 14:34:08 CST 2019" }.`  At build time we could include a process that reads from this "blogpost_upated.json" and updates the `updated_at` blogpost of each blogpost.

We could youse what you proposed to get the filename and date and write the result to the `blogpost_upated.json`

```bash
git log -n 1 --pretty=format:%H -- <FILE PATH> (returns the commit id of the file)
git show -s --format=%ci <COMMIT ID> (returns the last commit date of the file)
```

The solution is cumbersome and ugly... but given that we don't have a git to provide the dates and we can't control the environment in which the blog is deployed i think it's reasonable solution.

**Comment by etagwerker** [comment]
@bronzdoc I believe the problem is this: The blog is a gem dependency of ombulabs.com and fastruby.io. When deploying the application, there is no way of seeing `git history` for said gem dependency. Git history is not packaged in a gem, so that is why we can't access git history. 

One way to solve this problem would be to somehow fetch git history in the `release` phase in Heroku. That way we can get the git history of the blog, then we can calculate the "Updated at:" value per blog post. 

To sum it up, it would work like this: 

1. Heroku receives push
2. Heroku starts deployment phase 
3. Our application could fetch `git history` for the `blog` project 
4. Our application generates the HTML based on the content inside the `blog` dependency

This way, our application will know the history of the `blog` project. 

Do you see any problems with a solution like that one?
